src/Components/Blueprints/BlueprintsSideBar.tsx

```diff
@@ -61,7 +81,7 @@ const BlueprintsSidebar = ({
         <StackItem>
           <SearchInput
```

Comment:
A nice a addition for the search input is adding the `resultsCount` prop for rendering a counter badge, i.e
![Screenshot 2024-01-29 at 14 42 54](https://github.com/RedHatInsights/image-builder-frontend/assets/11807069/687a61ac-140f-4db7-a6a1-a53b86b5151d)

Reply:
I've used `count blueprints` it feels more user friendly, but kinda long, so I'm not strongly for that, happy to go just with the number if you'd think it's better ;)

Reply:
agree, feels friendly and verbose, but a bit too long IMHO